linode/manager
# refactor: Clean up Account Settings Object Storage

## Description 📝

- Refactors the Object Storage section of the https://cloud.linode.com/account/settings page 🔁 
- Adds a React Query mutation called `useCancelObjectStorageMutation` so we don't have to manage our own loading and error states 🧼 😃 
- Renames the component from `EnableObjectStorage` to  `ObjectStorageSettings` ✏️ 
  -  `EnableObjectStorage` was very confusing. This component's main purpose is to allow users to _**disable**_ object storage
- Adds a new toast for when you successfully disable object storage 🍞 
- Updates the unit tests to account for changes 🧪 

## Preview 📷

> [!note]
> UI should not look any different. All changes were to just clean up the code and use React Query for the mutation

https://github.com/user-attachments/assets/d1c11ff9-cbe8-40fe-8665-d0a236011554

## How to test 🧪

- Test the disable Object Storage flow on http://localhost:3000/account/settings
  - The `enable-object-storage.spec.ts` covers this flow so ensure that test passes ✅ 

## As an Author I have considered 🤔
